pyproject.toml

```
@@ -75,6 +75,7 @@ dependencies = [
     "pulsar-galaxy-lib>=0.15.10",
     "pycryptodome",
     "pydantic[email]>=2.7.4",  # https://github.com/pydantic/pydantic/pull/9639
+    "pydicom; python_version >= '3.10'",
```

Unless you need a specific version of pydicom, the more flexible solution is:

```suggestion
    "pydicom",
```

and let `./lib/galaxy/dependencies/update.sh -p pydicom` resolve it for you.

---

Thanks, done in 3b96806556737039d7998c0271d2d534b81ad603, worked!